zend_abstract_interface/config/tests/default.cc

```diff
@@ -0,0 +1,84 @@
+extern "C" {
+#include "config_test_helpers.h"
+
+#include "config/config.h"
+#include "zai_sapi/zai_sapi.h"
+#include "zai_sapi/zai_sapi_extension.h"
+#include "ext_zai_config.h"
+}
+
+#include "zai_tests_common.hpp"
+
+TEST_DEFAULT("bool", {
+    REQUEST_BEGIN()
```

Comment:
this seems repetitive, should be part of the TEST_DEFAULT?

Reply:
Not without another few macros, so I left it ... but did define TEST_DEFAULT (and others)